fc/protocol.py

```diff
@@ -151,57 +152,58 @@ def __init__(self, proto_file, indent_level=0):
 
         # 4. Any number of ``import`` statements (again, no section)
         # https://chaste.cs.ox.ac.uk/trac/wiki/FunctionalCuration/ProtocolSyntax#Importsofotherprotocols
-
-        # TODO: Not sure what this dict contains
+        # Maps an import 'name' prefix to a :class:`Protocol` instance.
         self.imports = {}
 
         # 5. The ``library`` section
         # https://chaste.cs.ox.ac.uk/trac/wiki/FunctionalCuration/ProtocolSyntax#Library
         # Can contain assignment statements (``var = expr``), function
         # assignment statements (``var = lambda(...)``), or assertions
         # (``assert cond``).
-
-        # TODO: Do all of these end up in this list?
-        # TODO: Who checks the assertions?
+        # These statements will be executed when the protocol is run.
         self.library = []
 
         # 6. The ``units`` section
         # https://chaste.cs.ox.ac.uk/trac/wiki/FunctionalCuration/ProtocolSyntax#Physicalunitdefinitions
-        # TODO: Where do these end up?
+        self.units = UnitStore()
```

Comment:
would have to be `self.units = UnitStore(model.units)`, I guess?

Reply:
Except we're setting up the protocol first, so actually when parsing the model we want to tell it to use the protocol's store?

Reply:
Guess so!